org.gnome.Fractal.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "org.gnome.Fractal",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "fractal",
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--talk-name=org.freedesktop.secrets",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "build-options" : {
+        "env": {
+            "CARGO_HOME": "/run/build/Fractal/cargo"
+        }
+    },
+    "modules": [
+        {
+           "name": "Fractal",
+           "buildsystem": "simple",
+           "build-commands": [
+               "source /usr/lib/sdk/rust-stable/enable.sh && ./configure --prefix=/app && make && make install"
+            ],
+           "sources": [
```

Comment:
Forgot to install the appdata file, you can do it from `post-install`.

Reply:
The appdata file now is installed by the app make install so I've removed from this branch.